mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -78,7 +78,7 @@ def __remote_init(self):
 
         # Remote DUT connection, flashing and reset...
         try:
-            self.__remote_disconnect()
+            #self.__remote_disconnect()
```

Comment:
it should be safe to remove whole line. it was originally workaround for one bug which is fixed long time ago already..